app/enums/TaskState.tsx

```diff
@@ -0,0 +1,7 @@
+enum TaskState {
+  unscheduled,
+  inProgress,
+  done
+}
+
+export { TaskState }
```

Comment:
Why should we use a enum here?

```suggestion
export const TaskState = {
	unscheduled: {text: "unscheduled", label: 1},
	inProgress: {text: "in progress", label: 2},
	done: {text: "done", label: 3},
}
```

So just use:
```ts
const doneState = TaskState.done;
console.log(doneState.text, doneState.label)
```

Reply:
Is also possible.